### ISSUE TITLE
Prisma: Create Get Endpoints for Applications, application flagged sets, and users

### ISSUE THREAD

**Issue by YazeedLoonat** [bloom-housing/bloom#3504]
blocked by: https://github.com/bloom-housing/bloom/pull/3484

applications: https://github.com/bloom-housing/bloom/pull/3494 (on hold)
users: https://github.com/bloom-housing/bloom/pull/3546 (out for review)
